docs/framework/security/building-my-first-claims-aware-aspnet-web-app.md

```diff
@@ -27,11 +27,11 @@ author: "BrucePerlerMS"
   
  STS is a service that issues tokens upon successful authentication. Microsoft offers two industry standard STS’s:  
   
-- [Active Directory Federation Services (AD FS) 2.0](https://go.microsoft.com/fwlink/?LinkID=247516)
```

Comment:
This entire folder (framework/security) is being deleted at #14590 and moved to the archive. It would be better if we can roll back the changes for this entire folder.